lib/classes/PluginManager.js

```diff
@@ -54,9 +54,9 @@ class PluginManager {
     let pluginProvider = null;
     // check if plugin is provider agnostic
     if (pluginInstance.provider) {
-      if (typeof pluginInstance.provider === 'string') {
+      if (_.isString(pluginInstance.provider)) {
         pluginProvider = pluginInstance.provider;
-      } else if (typeof pluginInstance.provider === 'object') {
+      } else if (_.isObject(typeof pluginInstance.provider)) {
```

Comment:
The typeof is superfluous and should be removed: `_.isObject(pluginInstance.provider)`

Reply:
@HyperBrain fixed, refactoring type.